Models/Spectrometer.cs

```diff
@@ -201,15 +204,15 @@ public async Task<bool> initAsync(Dictionary<string, ICharacteristic> characteri
         integrationTimeMS = 400;
         gainDb = 8;
 
-        verticalROIStartLine = eeprom.ROIVertRegionStart[0];
-        verticalROIStopLine = eeprom.ROIVertRegionEnd[0];
+        // verticalROIStartLine = eeprom.ROIVertRegionStart[0];
+        // verticalROIStopLine = eeprom.ROIVertRegionEnd[0];
```

Comment:
I commented-out the vertical ROI stuff because I think it's no longer in the BLE API; I think Ram will have to re-create these opcodes using the new generic Characteristic, and though that shouldn't be a big deal, it hasn't been done yet.